terraform/outputs.tf

```diff
@@ -10,15 +10,15 @@ output "app_name" {
 
 output "fiveg_nrf_endpoint" {
   description = "Name of the endpoint used to integrate with the NRF."
-  value = "fiveg-nrf"
+  value       = "fiveg_nrf"
```

Comment:
Didn't we decide to go with `-` instead of `_`?

Reply:
I think we decided to go with "fiveg_nrf". The reason is only 2 charms are using "fiveg-nrf" as the interface name. What I recall from our discussion, we can follow the convention with one that most of them have. In the `charm name convention guide`, there is not a rule for this.

Reply:
Now I will open PRs to fix interface names in AMF and NRF.

Reply:
OK